recipe/meta.yaml

```
@@ -1,45 +1,50 @@
 {% set name = "jupyterhub-ldapauthenticator" %}
-{% set version = "1.3.2" %}
-{% set sha256 = "758081bbdb28b26313bb18c9d8aa2b8fcdc9162e4d3ab196c626567e64f1ab8b" %}
+{% set version = "2.0.2" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/jupyterhub_ldapauthenticator-{{ version }}.tar.gz
+  sha256: c08cecd9bd996ab523c810d45204bb4a5eaac7adae38d3ef05eebc58fffc7a87
 
 build:
-  noarch: python
   number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  skip: true  # [py<39]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   host:
-    - python >=3.5
+    - python
     - pip
     - setuptools
   run:
-    - python >=3.5
-    - jupyterhub
-    - ldap3
-    - tornado
+    - python
+    - jupyterhub >=4.1.6
+    - ldap3 >=2.9.1
     - traitlets
 
 test:
   imports:
     - ldapauthenticator
+  requires:
+    - pip
+  commands:
+    - pip check
```

https://github.com/jupyterhub/ldapauthenticator/tree/2.0.2/ldapauthenticator/tests
```suggestion
    - pip check
    - pytest --pyargs ldapauthenticator.tests
```

---

Done. Thanks!

---

The source distribution on PyPI doesn't have tests https://inspector.pypi.io/project/jupyterhub-ldapauthenticator/2.0.2/packages/52/43/3f8b33857020e17737e75727b01bf75a3a626da55a61dcced8212744078c/jupyterhub_ldapauthenticator-2.0.2.tar.gz/